src/components/ViewableTrackList.js

```diff
@@ -16,4 +16,4 @@ const TrackListView = props => {
   );
 };
 
-export default TrackListView;
+export default ViewableTrackList;
```

Comment:
I actually would've gone the other direction. Dumb component gets dumb name (TrackList) while container gets a qualifier added to the dumb name (FilteredTrackList).

YMMV.